services/system/AccountSys/ui/tsconfig.json

```diff
@@ -2,15 +2,12 @@
     "compilerOptions": {
         "target": "ESNext",
         "useDefineForClassFields": true,
-        "lib": [
-            "DOM",
-            "DOM.Iterable",
-            "ESNext"
-        ],
+        "lib": ["DOM", "DOM.Iterable", "ESNext"],
         "allowJs": false,
         "paths": {
-            "/common": [
-                "../../../user/CommonSys/common"
+            "/common": ["../../../user/CommonSys/common"],
```

Comment:
This doesn't feel right. Is this for static assets like fonts? Wouldn't that be a common/public folder or something? given loader and rpc code are both in packages/ subdir?

Reply:
This was just a formatting change. Instead of digging into every app to see what _else_ they're consuming from the `common` directory, I focused more narrowly--on making sure they're not trying to consume the old `rpc.mjs` and `keyConversions.mjs`... and that they're instead consuming the new `rpc.ts`, `keyConversions.ts` (in dev) and `common-lib.js` (at runtime).

These other, less specific references to `common` will be cleaned up when we start getting these apps working again.